src/main/java/com/projects/oop/abstracts/Panther.java

```diff
@@ -0,0 +1,11 @@
+package com.projects.oop.abstracts;
+
+public class Panther extends Animal {
+    void furColour(){
+        System.out.println("A panther has black fur");
```

Comment:
all methods should return string instead of println so that the caller can do what it wants with the string